cmd/api/routers/routers.go

```diff
@@ -19,7 +23,7 @@ type Controller struct {
 func (c *Controller) GetAllResources(context *gin.Context) {
 	group := context.Param("group")
 	version := context.Param("version")
-	resource := getResourceKindFromPlural(context.Param("resourceType"))
+	resource := getAPIResourceKind(context)
```

Comment:
Shouldn't this really be called `kind`?

Reply:
No, I meant like this:
```
kind := getAPIResourceKind(context)
```

Reply:
yes, that's the change that I did, don't you see it updated in the `Files Changed` section of this PR?
Note that this thread keeps the outdated code (there is the `outdated` label on it)